### PR TITLE
5.2: General topology updates: Part 2

### PR DESCRIPTION
**Pull request by nadinequinr3**

### CODE REVIEW

File: content/en/platform/corda/5.2/developing-applications/ledger/flows/_index.md [hunk 1]
```diff
@@ -55,14 +56,18 @@ For example, Alice is a participant on the network and wishes to agree a ledger
...
+
+Virtual nodes can have multiple active flows running simultaneously. Corda serializes flows to an out-of-process state manager. If the flow logic only involves synchronous HTTP calls when contacting other workers, the flow itself will not be serialized. This allows flows to be active for long periods of time, even during interruptions such as node restarts and upgrades. Instead of waiting for the flow to become unblocked, the node immediately starts work on any other scheduled flows, returning to the original flow at a later date. A flow will suspend whenever it needs to perform an operation using platform APIs. Flows can survive a failover event, allowing them to migrate to another {{< tooltip >}}flow worker{{< /tooltip >}} process if required.
```
[the remark this answers is not in the document]
Comment (nadinequinr3) [reply content/en/platform/corda/5.2/developing-applications/ledger/flows/_index.md]: I've pushed an edit to make it vaguer. Let me know what you think...

File: content/en/platform/corda/5.2/key-concepts/cordapp-dev/building/_index.md [hunk 2]
```diff
@@ -125,6 +125,11 @@ Secondly, a long wait increases the probability of being interrupted.
 
 In order to avoid these long waits, and also to facilitate building fault tolerant workflows, Corda uses a suspend/resume model.
 In practice, every time one of these “out-of-process” operations happens, such as, signing, persisting, or peer-to-peer communications, the flow should suspend and a checkpoint created.
+
+{{< note >}}
+Corda does not create checkpoints for synchronous HTTP calls on the out-of-process state manager.
```
[the remark this answers is not in the document]
Comment (nadinequinr3) [reply content/en/platform/corda/5.2/key-concepts/cordapp-dev/building/_index.md:130]: Do you think it's ok just to remove the note? The "In practice," in the proceeding question might be enough to suggest that checkpoints are not always created?

[the remark this answers is not in the document]
Comment (jujoramos) [reply content/en/platform/corda/5.2/key-concepts/cordapp-dev/building/_index.md:130]: Yeah, that sounds good.

File: content/en/platform/corda/5.2/developing-applications/ledger/flows/_index.md [hunk 3]
```diff
@@ -55,14 +56,18 @@ For example, Alice is a participant on the network and wishes to agree a ledger
...
+
+Virtual nodes can have multiple active flows running simultaneously. When appropriate, Corda serializes flows to an out-of-process state manager. This allows flows to be active for long periods of time, even during interruptions such as node restarts and upgrades. Instead of waiting for the flow to become unblocked, the node immediately starts work on any other scheduled flows, returning to the original flow at a later date. A flow will suspend whenever it needs to perform an operation using platform APIs. Flows can survive a failover event, allowing them to migrate to another {{< tooltip >}}flow worker{{< /tooltip >}} process if required.
```
Comment (jujoramos) [inline content/en/platform/corda/5.2/developing-applications/ledger/flows/_index.md:73]: > A flow will suspend whenever it needs to perform an operation using platform APIs.

The problem here is that it _**literally depends on the platform API used by the Corda Application**_... some calls might suspend the flow (asynchronous) and some won't (synchronous). Unless we include the full list in the documentation (or Javadocs, for what's worth), customers won't have any idea regarding when the suspension will happen.

**Pull request merged**